### ISSUE TITLE
Onhover Insert Image help text is wrong

### ISSUE THREAD

**Issue by tahnok** [dellsystem/wikinotes#154]
When you hover over the Insert Image button (the camera) it says: "Image. [Alt text](path/to/image.png)". This isn't the right syntax for markdown images. The correct text should be: "Image. ![Alt text](path/to/image.png)"

Button inserts the right markdown
